src/test/java/com/ccs/inventorymanagement/ClothingServiceTest.java

```diff
@@ -0,0 +1,253 @@
+package com.ccs.inventorymanagement;
+
+import com.ccs.inventorymanagement.domain.Clothing;
+import com.ccs.inventorymanagement.domain.Item;
+import com.ccs.inventorymanagement.repo.ClothingEntity;
+import com.ccs.inventorymanagement.repo.ClothingRepository;
+import com.ccs.inventorymanagement.service.ClothingService;
+import io.r2dbc.spi.R2dbcBadGrammarException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.time.Instant;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class ClothingServiceTest {
+
+    @Mock
+    ClothingRepository clothingRepository;
+
+    @InjectMocks
+    ClothingService clothingService;
+
+    @Test
+    @DisplayName("GET Find All: Should return all clothing found in the database")
+    public void shouldReturnAllClothing() {
+
+        // Given
+        final UUID id = UUID.randomUUID();
+        ClothingEntity entity = mock(ClothingEntity.class);
+
+
+        // When
+        when(clothingRepository.findAll())
+                .thenReturn(Flux.just(entity));
+        when(entity.getId())
+                .thenReturn(id);
+
+
+        // Then
+        clothingService.findAll()
+                .as(StepVerifier::create)
+                .expectNextCount(1)
+                .verifyComplete();
+        verify(clothingRepository, times(1)).findAll();
```

Comment:
This doesnt really test much, just that the repo is invoked. You could expand this a bit by filling out your entity with data and asserting it. This would let us know any logic the service is performing is working as expected.